src/web/components/Counts.tsx

```diff
@@ -3,97 +3,25 @@ import { css } from 'emotion';
 
 import { border } from '@guardian/src-foundations/palette';
 
-import { ShareCount } from '@frontend/web/components/ShareCount';
-import { CommentCount } from '@frontend/web/components/CommentCount';
-import { useApi } from '@root/src/web/lib/api';
-import { formatCount } from '@root/src/web/lib/formatCount';
-import { joinUrl } from '@root/src/web/lib/joinUrl';
-
 type Props = {
-    ajaxUrl: string;
-    pageId: string;
-    pillar: Pillar;
-    commentCount?: number;
-    setIsExpanded: Function;
-};
-
-type ShareCountType = {
-    path: string;
-    share_count: number;
-    refreshStatus: boolean;
-};
-
+    children: React.ReactNode;
+}
 const containerStyles = css`
     display: flex;
     flex-direction: row;
     align-items: flex-start;
-`;
-
-const NumbersBorder = () => (
-    <div
-        data-testid="numbers-border"
-        className={css`
-            height: 40px;
-            margin-left: 4px;
-            margin-right: 4px;
-            border-left: 1px solid ${border.secondary};
-        `}
-    />
-);
-
-export const Counts = ({
-    ajaxUrl,
-    pageId,
-    commentCount,
-    pillar,
-    setIsExpanded,
-}: Props) => {
-    const shareUrl = joinUrl([ajaxUrl, 'sharecount', `${pageId}.json`]);
-    const { data: shareData, error: shareError } = useApi<ShareCountType>(
-        shareUrl,
-    );
-
-    if (shareError) {
-        window.guardian.modules.sentry.reportError(
-            shareError,
-            'share-comment-counts',
-        );
+    aside:not(:empty) + aside:not(:empty) {
```

Comment:
As we're relying on html structure and elements here, would it be worth adding classes and targeting them rather than `aside` to imply more of a contract?

Reply:
This css grew logically but I think you're right, looking at this fresh a simple class would be more readable